From 19d0ebe2d002410e72672ad830eca9d8a5ec23d9 Mon Sep 17 00:00:00 2001
From: Daniel Wagner <dwagner@suse.de>
Date: Tue, 23 Aug 2022 15:30:42 +0200
Subject: [PATCH] build: Only consider libtracecmd documentation

We are not interested to build the trace-cmd
documentation. Unfortuantly, we can't control this from the top level
Makefile, hence just disable it on source code level.

Signed-off-by: Daniel Wagner <dwagner@suse.de>
---
 Documentation/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Documentation/Makefile b/Documentation/Makefile
index ec364916d13f..89795defdba3 100644
--- a/Documentation/Makefile
+++ b/Documentation/Makefile
@@ -4,7 +4,6 @@ doc_dir:=$(src)/Documentation
 
 export doc_dir
 
-SUBDIR += trace-cmd
 SUBDIR += libtracecmd
 
 .PHONY: $(SUBDIR)
-- 
2.37.1

